File: JAVA DB/Spring Data/09. Hibernate Code First/src/main/java/Football_Betting_06/enums/ResultPrediction.java
package Football_Betting_06.enums;

public enum ResultPrediction {
    HOME_WIN("Home team Win"),
    AWAY_WIN("Away team Win"),
    DRAW("Draw game");

    private final String value;

    ResultPrediction(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: CSharp DB/Entity Framework Core/EF-Exercises/FootballBetting/FootballBetting.DomainClasses/ResultPrediction.cs
namespace FootballBetting.DomainClasses
{
    public class ResultPrediction
    {
        public int Id { get; set; }
        public WinDrawLost Prediction { get; set; }

    }
}
//•	ResultPrediction – Id, Prediction (possible values - Home Team Win, Draw Game, Away Team Win)